components/advanced_create_comment/advanced_create_comment.tsx

```diff
@@ -413,9 +413,13 @@ class AdvancedCreateComment extends React.PureComponent<Props, State> {
         });
     }
 
-    toggleEmojiPicker = () => {
-        this.setState({showEmojiPicker: !this.state.showEmojiPicker});
-    }
+    toggleEmojiPicker = (e?: React.MouseEvent<HTMLButtonElement, MouseEvent>): void => {
+        if (e) {
+            e.stopPropagation();
+        }
+        const showEmojiPicker = !this.state.showEmojiPicker;
+        this.setState({showEmojiPicker});
```

Comment:
This should probably have been done as the functional version of setState before
```suggestion
        this.setState((state) => {showEmojiPicker: !state.showEmojiPicker});
```